docs/events/init.md

```diff
@@ -0,0 +1,58 @@
+# INIT
+
+**async | master**
+
+The `INIT` event is triggered before the `run` or `readTests tasks are executed. The event handler can be asynchronous: in this case, the tasks will start executing only after the _Promise_ returned by the event handler is resolved. The event is triggered only once, regardless of how many times the tasks are executed.
```

Comment:
backticks mismatch

Also it's not clear what tasks will start executing. I'd just write "Event handler may return a Promise. The execution will proceed only when it's resolved."